universal-application-tool-0.0.1/app/services/question/QuestionDefinition.java

```diff
@@ -75,6 +79,10 @@ public String getQuestionText(Locale locale) {
     throw new RuntimeException("Locale not found: " + locale);
   }
 
+  public ImmutableMap<Locale, String> getQuestionText() {
+    return questionText;
+  }
+
```

Comment:
question: is this being added to support serialization?

Reply:
yeah it looks like it is automatically looking for getter methods for serialization (see the @JsonIgnore annotations used to ignore the getters we don't want for serialization). I'll add a comment for this.